slurm/pi_aggregation.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
```

Comment:
Want to add some docstrings here, explaining our general logic so that someone reading can learn why and copy it?

Reply:
Do you mean the whole script or the shebang line?

Reply:
I added a docstring for the ``pi_aggregation.py`` script.